content/en/about/board/_index.md

```diff
@@ -24,37 +24,43 @@ aliases:
+Previously a Director of Engineering for Deloitte, Matt consulted with technology executives on engineering strategy and has also worked with RWE Supply and Trading, BP, Shell and National Australia Bank. With over 18 years transformation experience in the UK, Europe and Australia, Matt has a passion for mentoring engineers towards engineering excellence and is an active supporter of technical and local communities.
 {{< /board-member >}}
 
-{{< board-member name="Katie Schueths" image="/images/about/Katie_Schueths.jpg" >}}
-Katie Schueths leads the InnerSource program at Analog Devices Inc (ADI). She is building an InnerSource framework at ADI to help improve collaboration, code reuse, and documentation across their engineering organization. Previously, she started the InnerSource program at Indeed. She is part of the InnerSource Commons Marketing Group, helping to plan monthly community calls and events. In her free time, Katie enjoys running operational logistics for volunteer-led community art events.
+{{< board-member name="Katie Schueths" title="Assistant Secretary" image="/images/about/Katie_Schueths.jpg" >}}
+Katie Schueths leads the InnerSource program at Analog Devices Inc (ADI). She is building an InnerSource framework at ADI to help improve collaboration, code reuse, and documentation across their engineering organization. Previously, she started the InnerSource program at Indeed. She is part of the InnerSource Commons Marketing Group, helping to plan monthly community calls and events. In her free time, Katie enjoys running operational logistics for volunteer-led community arts events.
 {{< /board-member >}}
 
-{{< board-member name="Matt Cobby" title="Secretary" image="/images/about/Matt_Cobby.jpg" style="bg-light" >}}
-Matt Cobby is Director of Engineering at Deloitte where he specializes in helping teams with their engineering practices, developer experience, and platform & product engineering.
-As a member of the InnerSource Foundation he has run a large scale InnerSource program and now seeks to inspire teams to seek the many benefits of an Open Source style of collaboration inside their own organization.
-With over 18 years transformation experience in the UK, Europe & Australia, he has a passion for mentoring engineers and is an active supporter of technical & local communities that inspire people.
+{{< board-member name="Tom Sadler" title="Treasurer" image="/images/about/Tom_Sadler.jpg" style="bg-light" >}}
+Tom Sadler is a Software Engineering Team Lead for BBC iPlayer and Sounds, working in the connected TV space and focusing on cross-team collaboration, including InnerSource. Tom has been involved with InnerSource Commons since 2019, speaking at multiple community events. He is a Trusted Committer on the InnerSource Learning Path project, and the Lead on publishing the Learning Path to innersourcecommons.org.
```

Comment:
```suggestion
Tom Sadler is a Principal Software Engineer for BBC iPlayer and Sounds, working in the connected TV space and focusing on cross-team collaboration, including InnerSource. Tom has been involved with InnerSource Commons since 2019, speaking at multiple community events. He is a Trusted Committer on the InnerSource Learning Path project, and the Lead on publishing the Learning Path to innersourcecommons.org.
```